linux_kernel_modules/cypwifi/scripts/cywifi.sh

```diff
@@ -107,13 +118,16 @@ cy_wifi_start() {
 cy_wifi_stop() {
 	ifconfig | grep wlan0 >/dev/null
         # TODO FIX:  For now we have commented rmmod's and kicking the Cypress chip
+	# The sdhci-msm crashes on modprobe -r 
 	#if [ $? -eq 0 ]; then
 	   #ifconfig wlan0 down
 	#fi
 	#rmmod /legato/systems/current/modules/brcmfmac.ko || exit 127
 	#rmmod /legato/systems/current/modules/brcmutil.ko || exit 127
 	#rmmod /legato/systems/current/modules/cfg80211.ko || exit 127
 	#rmmod /legato/systems/current/modules/compat.ko || exit 127
+	#modprobe -r sdhci-msm
```

Comment:
Should all these rmmods go away now that we are using modprobe for this?

Reply:
I would love to use modprobe for all kernel modules. Alas, we have 2 different unshared dependency graphs co-existing. The kernels depmod system that is used by the kernel at build time only at SWI. And the Legato dependency module graph system kmod. And finally, since Cypress is built in the Cypress build system we are stuck with rmmod. I am tempted one of these days to get depmod on the target & have Modules.dep be writable (maybe it is or isn't in squashfs). Then add the Cypress modules on the target & rerun depmod. Anyways, short answer disparate systems issue. Later we can fix

Reply:
Ah, sorry.  I didn't read that very well.  Rushing too much.  I see you're just adding one more to the list, in hopes that someday the QCT driver will be fixed such that these can be safely unloaded.